EIPS/eip-5553.md

```diff
@@ -0,0 +1,260 @@
+
+### ledger() function
+returns the address of the registry or registrar contract, or an EOA account that initialized the work and associated royalty tokens. A work might be registered in multiple places by different actors for different purposes, and this enables market participants to discover which registry mechanism is the parent of the work and might have special access rights to manage the work.
+
+## Motivation
+There is no accepted standard mechanism to license a work, or to represent it, except using traditional NFTs. But regular NFTs only represent a collectible item use case, and cannot easily represent more complicated use cases of licensing a work for different types of uses.
+To enable such mechanisms, we need a more robust mechanism for :
+1) Declaring that a work exists, SEPARATE from its purchase ability
+2) Declaring possibly multiple interested parties to be paid for such work 
+
+for #1 no standard exists today.
+For #2 we only have regular splits standards based on NFT purchases, or through mechanisms like 0xsplits. While these are a great headstart, they do not contain the ability to name multiple types of collaboration participants, and more importantly, do not contain an easy way to manage the splits in a way that feels natural.
+
+With this proposal, registration is taken care of, and splits are taken care of by defining the splits as an open array of EIP-20 tokens that represent split percentages. Moving splits to various participants will be as simple as sending an EIP-20 token from one wallet to another. discovering split movements is as simple as checking EIP-20 transfer events.
+
+In the future, we can build full payout mechanisms based on these simple split tokens, and licensing standards on top of those payout mechanisms, which are not possible today due to the complexity of splits management in the traditional web2 world.
+
+## Specification 
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and
+"OPTIONAL" in this document are to be interpreted as described in
+RFC 2119.
+
+**contracts that want to represent a work on chain MUST implement [EIP-721](./eip-721.md) AND this Proposal**
+
+Implementers of this standard **MUST** have all of the following functions:
+
+```solidity
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+import '@openzeppelin/contracts/interfaces/IERC165.sol';
+
+
+///
+/// @dev Interface for Simple Works Registration Standard
+///
+interface IWorksRegistration is IERC165 {
+    
+    /// @notice Called with the new uri to an updated metadata file
+    /// @param _newUri - the uri pointing to a metadata file (file standard is up to the implementer)
+    /// @param _newFileHash - The hash of the new metadata file, for future reference and verification
+    function changeMetadataURI(string memory _newUri, string memory _newFileHash) external ;
```

Comment:
[Editorial, Peer] Consider add an event when URL changed?

Reply:
There is already a MetadataChanged event specified:
    event MetadaDataChanged(address byAddress,string oldURI, string oldFileHash, string newURI, string newFileHash);